lib/internal/Magento/Framework/Phrase/__function.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Create value-object \Magento\Framework\Phrase
+ *
+ * @return \Magento\Framework\Phrase
+ */
+function __()
```

Comment:
Wasn't idea to put this function in Magento/Framework/Phrase/ - `\Magento\Framework\Phrase` namespace?

Reply:
I don't think that was proposed so far. Wouldn't namespacing the function mean all existing templates would need to be adjusted?

Reply:
Let's wait @orlangur for review